lib/seo_meta.rb

```diff
@@ -34,8 +34,13 @@ def is_seo_meta(options = {})
     end
 
     # Let SeoMetum know about the base
-    ::SeoMetum.send :belongs_to, self.name.underscore.gsub('/', '_').to_sym,
-                    :class_name => self.name
+    if ActiveRecord::VERSION::STRING >= '5.0.0'
+      ::SeoMetum.send :belongs_to, self.name.underscore.gsub('/', '_').to_sym,
+                      class_name: self.name, optional: true
```

Comment:
Finally i've deprecated all Rails version before Rails 5 because the conditional code was needed for Rails < 5.0 and also because seo_meta 2.0.x worked well with Rails < 5.0